example/src/main/java/com/rockerhieu/rvadapter/endless/example/adapter/SimpleStringAdapter.java

```diff
@@ -55,13 +55,13 @@ public int getItemCount() {
     }
 
     public void appendItems(List<String> items) {
-        int count = getItemCount();
         mValues.addAll(items);
-        notifyItemRangeInserted(count, items.size());
+        notifyItemRangeInserted(getItemCount() + 1, items.size());
```

Comment:
Let me investigate this change, see the reason why it was crashing without `+1`.

10 mins...